00_Base/package.json

```diff
@@ -37,5 +36,6 @@
     "tslog": "^4.9.2",
     "uuid": "^9.0.0",
     "zod": "^3.22.2"
-  }
+  },
+  "workspace": "../"
```

Comment:
Ah got it. So in theory Base could have this omitted. The connection I hadn't made was how the dependancies work with each other. E.g. Data depending on Base and setting the workspace to `../` But since you need it to be part of the workspace, I understand.

Reply:
I would omit it the other way, if you call npm commands from the root then you can omit it in the children. Including it in both I would say allows you to run npm commands from both the parent and the child module folders